connector-template/mod.rs

```diff
@@ -168,7 +168,7 @@ impl
         &self,
         req: &types::PaymentsSyncRouterData,
         connectors: &settings::Connectors,
-    ) -> CustomResult<Vec<(String, String)>, errors::ConnectorError> {
+    ) -> CustomResult<Vec<\(String, services::request::header::Value\)>, errors::ConnectorError> {
```

Comment:
why we need escape sequence here? If needed so, why at line:46 is not needed?

Reply:
I'll remove it